### PR TITLE
F/wyy 160 56935 cns 534 73902

### PR DESCRIPTION
**Pull request by juuliabellcom**

### CODE REVIEW

File: public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module [hunk 1]
```diff
@@ -73,10 +73,10 @@ function os2web_accessibility_frontpage($nid = '') {
     $print_node_id = $nid_setting;
   }
 
-  $print = print_controller($print_node_id, $query, $cid, PRINT_HTML_FORMAT);
-
+  $print =  print_controller($query, PRINT_HTML_FORMAT, $cid);
```
Comment (stanbellcom) [inline public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module]: `PRINT_HTML_FORMAT` is not defined anywhere, so it's NULL.
@juuliabellcom are you sure the functionality works as expected?

`$cid` is also not defined, but it's a nullable param. For clarify it's better to not include it at all here.

File: public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module [hunk 2]
```diff
@@ -165,13 +166,14 @@ function os2web_accessibility_theme() {
       'variables' => array(),
     ),
     'os2web_accessibility' => array(
-      'variables' => array(
+      'arguments' => array(
```
Comment (stanbellcom) [inline public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module]: why did we need to rename it?
`variables` is default key for the theme array.

File: public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module [hunk 3]
```diff
@@ -265,7 +267,7 @@ function os2web_accessibility_site_map_menus($true = FALSE) {
 function os2web_accessibility_menu_link_add_handicap($tree = array()){
   foreach ($tree as $i =>$link) {
     foreach ($link as $key => $value) {
-      foreach ($value as $name => $v) {
+      foreach (@$value as $name => $v) {
```
Comment (stanbellcom) [inline public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module]: it confusing why we need a suppress sign here.
Maybe add a comment or make the code more readable?

File: public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module [hunk 4]
```diff
@@ -165,13 +166,14 @@ function os2web_accessibility_theme() {
...
     'os2web_accessibility' => array(
-      'variables' => array(
+      'arguments' => array(
         'handicap' => array(),
         'type' => PRINT_HTML_FORMAT,
         'node' => NULL,
         'menu' => NULL,
         'right_blocks' => NULL
       ),
+      'path' => drupal_get_path('module', 'os2web_accessibility'),
```
Comment (stanbellcom) [inline public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module]: why is this line needed? Where the path is used?

File: public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module (hunk 1 above)
Comment (juuliabellcom) [reply public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module]: @stanbellcom fixed https://github.com/syddjurs/syddjurs.dk/pull/169/commits/2670201384e129a508e11056f9de471e934b6e1d#diff-451abd137472af0eebe44aa8a5c4075cR76

File: public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module (hunk 2 above)
Comment (juuliabellcom) [reply public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module]: @stanbellcom I revert these changes, it was my fault

File: public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module (hunk 3 above)
Comment (juuliabellcom) [reply public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module]: @stanbellcom if menu punkt have no child punkts when "below" item is FALSE, and warning appeared. I changed code a little https://github.com/syddjurs/syddjurs.dk/pull/169/commits/2670201384e129a508e11056f9de471e934b6e1d#diff-451abd137472af0eebe44aa8a5c4075cR271

File: public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module (hunk 4 above)
Comment (juuliabellcom) [reply public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module]: @stanbellcom I delete this line
